.travis.yml

```diff
@@ -6,55 +6,24 @@
 #       significantly speed up CI times.                                #
 #########################################################################
 # Note: Setting `sudo` to `required` will add 20-50s to CI image boot time
-sudo: false
-dist: trusty
-services:
-  - docker
+sudo: true
+dist: bionic
 language: generic
 notifications:
   email:
     recipients:
 
 install:
-    # Pull in MoveIt's ROS Docker CI image (piping to `/dev/null` to hide output)
-    - DOCKER_IMAGE=moveit/moveit:melodic-ci
-    - echo "Pulling in Docker Image \"$DOCKER_IMAGE\""
-    - docker pull $DOCKER_IMAGE > /dev/null
-    # Tests default to false if not set
-    - RUN_BUILD=${RUN_BUILD:-"false"}
-    - RUN_TESTS=${RUN_TESTS:-"false"}
-    - RUN_COVERAGE=${RUN_COVERAGE:-"false"}
-    - RUN_FORMATTING_CHECKS=${RUN_FORMATTING_CHECKS:-"false"}
-    # Grab environment variables Codecov needs to run in the docker
-    - CODECOV_ENV=`bash <(curl -s https://codecov.io/env)`
-    # Run the CI script inside the docker container
-    # (set tests/checks to false by default)
-    - docker run $CODECOV_ENV
-      --env RUN_BUILD
-      --env RUN_TESTS
-      --env RUN_COVERAGE
-      --env RUN_FORMATTING_CHECKS
-      --env REPOSITORY_NAME
-      --env TRAVIS_PULL_REQUEST
-      --env TRAVIS_BRANCH
-      --volume $(pwd):/root/$REPOSITORY_NAME
-      $DOCKER_IMAGE
-      /bin/bash -c "/root/$REPOSITORY_NAME/travis_ci.sh"
+    - ./environment_setup/setup_software.sh
 
-env:
-  # Global Variables that are set at the start of every CI run
-  global:
-    # Get the name of this repository from the `TRAVIS_REPO_SLUG`
-    # ex. "UBC-Thunderbots/Software" would give "Software"
-    - REPOSITORY_NAME=$(basename $TRAVIS_REPO_SLUG)
-
-# Each entry here represents a different CI run
-# The variables here are uniquely set at the start of each different test
-matrix:
-  include:
-  - name: "ROS Melodic / Ubuntu 18.04 - Build and Test"
-    env: RUN_BUILD="true"
-         RUN_TESTS="true"
-         RUN_COVERAGE="true"
-  - name: "ROS Melodic / Ubuntu 18.04 - Check Formatting"
-    env: RUN_FORMATTING_CHECKS="true"
+script:
```

Comment:
without the matrix are we no longer running tests / formatting? or am I misunderstanding?

Reply:
_Holy, didn't even request a review yet_

Just playing around with stuff now, but yeah, will need to add coverage.

Reply:
Oh, I see github requested a review _for_ me.

Reply:
Sorry lol, if it shows up in my inbox I review it

Reply:
Haha same